Gems/Atom/Feature/Common/Code/Source/SkinnedMesh/SkinnedMeshOutputStreamManager.h

```diff
@@ -46,6 +46,8 @@ namespace AZ
             // SystemTickBus
             void OnSystemTick() override;
 
+            void EnsureInit() const;
+            void EnsureInit();
```

Comment:
I don't think we should be working around the constness of this function. EnsureInit is not const, it has a very real side effect. GetBufferAsset should no longer be const, since it can also have a side effect (when called for the first time).

Reply:
Addressed in dbc1f3bb62ec246115f6f7f22c9f766da58d9897